x/ibc/05-port/keeper/keeper.go

```diff
@@ -59,3 +63,19 @@ func (k Keeper) Authenticate(ctx sdk.Context, key capability.Capability, portID
 
 	return k.scopedKeeper.AuthenticateCapability(ctx, key, types.PortPath(portID))
 }
+
+// LookupModuleByPort will return the IBCModule along with the capability associated with a given portID
+func (k Keeper) LookupModuleByPort(ctx sdk.Context, portID string) (string, capability.Capability, bool) {
+	capOwners, ok := k.scopedKeeper.GetOwners(ctx, types.PortPath(portID))
+	if !ok {
+		return "", nil, false
+	}
+	if len(capOwners.Owners) < 2 {
+		return "", nil, false
+	}
+	// the module that owns the port, for now, is the first module that isn't "ibc"
+	// that owns the module
+	module := capOwners.Owners[1].Module
```

Comment:
Are we sure that the ordering will always be this way? Isn't it dependent on the module name?